apps/end-tenancy/models/upload.js

```diff
@@ -2,69 +2,77 @@
 'use strict';
 
 const url = require('url');
```

Comment:
out of scope for you but I would use destructure and import only the `URL` constructor from the `url` module 

```suggestion
const { URL } = require('url');
```
as parseUrl is now deprecated in favour of new URL